Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1330,10 +1330,6 @@ void TextEditor::leave_event(Core::Event&)
 {
```

Comment:
The commit message is the same as its description, that's redundant.
Also, please write a more concise message describing what's actually happening, and add what it fixes/enables to the description instead?

Reply:
Thank you! I've updated the commit messages, please check.